pkg/properties/properties_test.go

```diff
@@ -1,41 +1,12 @@
 package properties
 
 import (
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 )
 
-func TestParseLines_Empty(t *testing.T) {
-	lines := parseLines("")
-	require.Empty(t, lines)
-}
-
-func TestParseLines_Comment(t *testing.T) {
-	lines := parseLines("#key=val")
-	require.Empty(t, lines)
-}
```

Comment:
Even though we're getting rid of this function can we rewrite this test case? This is something we still want to support.

Reply:
In new PR https://github.com/confluentinc/cli/pull/2627, this function is not removed